funnel/loginproviders/linkedin.py

```diff
@@ -14,7 +15,7 @@
 
 class LinkedInProvider(LoginProvider):
     auth_url = 'https://www.linkedin.com/uas/oauth2/authorization?response_type=code&client_id={client_id}&scope={scope}&redirect_uri={redirect_uri}&state={state}'
-    token_url = 'https://www.linkedin.com/uas/oauth2/accessToken'
+    token_url = 'https://www.linkedin.com/uas/oauth2/accessToken'  # nosec
```

Comment:
Bandit detected this line as low severity hardcoded password for some reason. Had to add `# nosec`.